From 83848cf1ac84ad864cf014e0b9ccf8355da89c6a Mon Sep 17 00:00:00 2001
Message-ID: <83848cf1ac84ad864cf014e0b9ccf8355da89c6a.1697678664.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1697678664.git.sharlatanus@gmail.com>
References: <cover.1697678664.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 19 Oct 2023 00:54:42 +0100
Subject: [PATCH 14/19] gnu: go-github-com-google-go-cmdtest: Move to (gnu
 packages golang-check).

* gnu/packages/golang-check.scm (go-github-com-google-go-cmdtest): Move
from here...
* gnu/packages/golang.scm: ... to here.
---
 gnu/packages/golang-check.scm | 31 +++++++++++++++++++++++++++++++
 gnu/packages/golang.scm       | 29 -----------------------------
 2 files changed, 31 insertions(+), 29 deletions(-)

diff --git a/gnu/packages/golang-check.scm b/gnu/packages/golang-check.scm
index 82bcfcd28c..eb4f085bbf 100644
--- a/gnu/packages/golang-check.scm
+++ b/gnu/packages/golang-check.scm
@@ -10,6 +10,7 @@
 ;;; Copyright © 2022 Efraim Flashner <efraim@flashner.co.il>
 ;;; Copyright © 2023 Felix Lechner <felix.lechner@lease-up.com>
 ;;; Copyright © 2023 Hilton Chain <hako@ultrarare.space>
+;;; Copyright © 2023 Katherine Cox-Buday <cox.katherine.e@gmail.com>
 ;;;
 ;;; This file is part of GNU Guix.
 ;;;
@@ -101,6 +102,36 @@ (define-public go-github-com-frankban-quicktest
 tests.")
     (license license:expat)))
 
+(define-public go-github-com-google-go-cmdtest
+  (let ((commit "55ab3332a786118933ddf71544aae14951ba9bc5")
+        (revision "0"))
+    (package
+      (name "go-github-com-google-go-cmdtest")
+      (version (git-version "0.4.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/google/go-cmdtest")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32
+           "10kswvbdwissjb5mr0ys4b3ppxkxlpklqg7cr2z7rv21g2vwczbl"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "github.com/google/go-cmdtest"))
+      (propagated-inputs
+       (list go-github-com-google-renameio go-github-com-google-go-cmp-cmp))
+      (home-page "https://github.com/google/go-cmdtest")
+      (synopsis "Testing for your CLI")
+      (description
+       "The cmdtest package simplifies testing of command-line interfaces.  It
+provides a simple, cross-platform, shell-like language to express command
+execution.  It can compare actual output with the expected output, and can
+also update a file with new \"golden\" output that is deemed correct.")
+      (license license:asl2.0))))
+
 (define-public go-github-com-google-gofuzz
   (let ((commit "fd52762d25a41827db7ef64c43756fd4b9f7e382")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 8860ec50fe..5abe17bf3e 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -12860,35 +12860,6 @@ (define-public go-github-com-google-go-jsonnet
 implementation.")
     (license license:asl2.0)))
 
-(define-public go-github-com-google-go-cmdtest
-  (let ((commit "55ab3332a786118933ddf71544aae14951ba9bc5")
-        (revision "0"))
-    (package
-      (name "go-github-com-google-go-cmdtest")
-      (version (git-version "0.4.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://github.com/google/go-cmdtest")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "10kswvbdwissjb5mr0ys4b3ppxkxlpklqg7cr2z7rv21g2vwczbl"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/google/go-cmdtest"))
-      (propagated-inputs (list go-github-com-google-renameio
-                               go-github-com-google-go-cmp-cmp))
-      (home-page "https://github.com/google/go-cmdtest")
-      (synopsis "Testing for your CLI")
-      (description
-       "The cmdtest package simplifies testing of command-line interfaces.  It
-provides a simple, cross-platform, shell-like language to express command
-execution.  It can compare actual output with the expected output, and can
-also update a file with new \"golden\" output that is deemed correct.")
-      (license license:asl2.0))))
-
 (define-public go-github-com-google-safehtml
   (package
     (name "go-github-com-google-safehtml")
-- 
2.41.0

